Content.Server/Wires/WiresSystem.cs

```diff
@@ -526,6 +530,11 @@ private void OnMapInit(EntityUid uid, WiresComponent component, MapInitEvent arg
         if (component.WireSeed == 0)
             component.WireSeed = _random.Next(1, int.MaxValue);
 
+        if (TryComp<WiresPanelSecurityComponent>(uid, out var wiresPanelSecurity) &&
+            !string.IsNullOrEmpty(wiresPanelSecurity.SecurityLevel) &&
+            TryComp<ConstructionComponent>(uid, out var construction))
+            _construction.ChangeNode(uid, null, wiresPanelSecurity.SecurityLevel, true, construction);
+
```

Comment:
Extraneous whitespace

Reply:
Not really, otherwise it'll look like 'UpdateUserInterface(uid);' is part of the 'if' statement above